docs/HowToFAQ/FAQFailingPost.md

```diff
@@ -0,0 +1,30 @@
+---
+title: I can assert a condition right before a return, so why does the postcondition fail?
```

Comment:
I didn't immediately what the title of this one meant ("postcondition fail")

Reply:
Adjusted the question